src/qibo/quantum_info/metrics.py

```diff
@@ -638,14 +691,16 @@ def process_fidelity(channel, target=None, check_unitary: bool = False, backend=
 
     if target is None:
         # With no target, return process fidelity with Identity channel
-        fid = np.real(np.trace(channel)) / dim**2
-        fid = float(fid)
-        return fid
+        fidelity = np.real(np.trace(channel)) / dim**2
```

Comment:
I think this `fidelity` should be called differently since there is a function with the same name in this module.

Reply:
Done